mobile/native/src/channel_fee.rs

```diff
@@ -9,17 +9,21 @@ use anyhow::Context;
 use anyhow::Result;
 use bitcoin::Txid;
 use lightning_invoice::Invoice;
+use ln_dlc_node::node::rust_dlc_manager::subchannel::LNChannelManager;
 use ln_dlc_node::node::rust_dlc_manager::ChannelId;
+use ln_dlc_node::node::ChannelManager;
 use serde::Deserialize;
 use serde::Serialize;
 use std::str::FromStr;
 use std::sync::Arc;
 use std::sync::Mutex;
+use std::time::Duration;
 use tokio::runtime::Handle;
 
 #[derive(Clone)]
 pub struct ChannelFeePaymentSubscriber {
-    pub open_channel_tx: Arc<Mutex<Option<EsploraTransaction>>>,
+    pub open_channel_tx: Arc<Mutex<Option<(ChannelId, EsploraTransaction)>>>,
```

Comment:
🔧 Since we store two things now?

```suggestion
    pub open_channel_info: Arc<Mutex<Option<(ChannelId, EsploraTransaction)>>>,
```